libs/nx-vite/src/executors/serve/executor.ts

```diff
@@ -15,9 +15,12 @@ export default async function runExecutor(
     const tree = new FsTree(context.cwd, context.isVerbose)
     const appRoot = context.workspace.projects[context.projectName].root
 
+    const root = `${tree.root}/${appRoot}`
+    const configFile = options.configFile || `${root}/vite.config.js`
+
     const vite = execa(
         packageManager.exec,
-        ['vite', `${tree.root}/${appRoot}`, '--open'],
+        ['vite', root, '--config', configFile ,'--open'],
```

Comment:
If the config file doesn't exist, does that cause an issue?

Also, doesn't `vite.config.js` get picked up by convention?